dlgr/griduniverse/experiment.py

```diff
@@ -875,19 +900,35 @@ def rank(self, color):
             return 1
 
 
-class Food(object):
-    """Food."""
+class Object(object):
+    """Objects."""
     def __init__(self, **kwargs):
         self.id = kwargs.get('id', uuid.uuid4())
+        self.object_id = kwargs.get('object_id', 1)
+        self.name = kwargs.get('name')
```

Comment:
I don't think we should store these config properties which are the same for every object with the same type id. This will result in a lot of wasted RAM on large maps. We can just store the object properties dict as an attribute and then have properties that look these up, or just use `obj.type_config.get(prop)` or similar so all objects of the same type reference the same dict object (or we could not store it at all and have the GridWorld code look them up in the global object config). GU didn't store `food_reward` on `Food` and we probably shouldn't store `calories` on `Object`.